apps/smart-contracts/core/test/AllowedCallers.test.ts

```diff
@@ -11,37 +11,129 @@ describe('=> AllowedCallers', () => {
   let allowedCallers: AllowedCallers
   let deployer: SignerWithAddress
   let user: SignerWithAddress
+  let user2: SignerWithAddress
 
   beforeEach(async () => {
-    ;[deployer, user] = await ethers.getSigners()
+    ;[deployer, user, user2] = await ethers.getSigners()
     allowedCallers = await allowedCallersFixture()
   })
 
   describe('initial state', () => {
-    it('has no allowed accounts', async () => {})
+    it('has no allowed accounts', async () => {
+      expect(await allowedCallers.isCallerAllowed(deployer.address)).to.be.false
+    })
   })
 
   describe('# setAllowedCallers', () => {
-    it('reverts if array length mismatch', async () => {})
+    it('reverts when callers.length != allowed.length', async () => {
```

Comment:
```suggestion
    it('reverts if array length mismatch', async () => {
```
Please follow the testcase convention already commented on here => https://github.com/prepo-io/prepo-monorepo/pull/571

I'm not going to comment on everyone, but you get the gist.